docs/Consensus-Protocols/Clique.md

```diff
@@ -93,9 +93,15 @@ The JSON-RPC methods to add or remove signers are:
 * [clique_getSigners](../Reference/Pantheon-API-Methods.md#clique_getsigners)
 * [clique_discard](../Reference/Pantheon-API-Methods.md#clique_discard)
 
+!!! important
+    Adding or removing a signer is subject to agreement by existing signers. Meaning the command to add or remove a signer must
```

Comment:
I'd reword the first sentence to be active: 
A majority of existing signers must agree on adding or removing a signer. 

And be more specific about what _the command_ is: 
That is, `clique_proposeValidatorVote` must be executed specifying the signer to add or remove on the majority ...

Reply:
Agreed. Fixed it here and for IBFT.Agreed. Fixed it here and for IBFT.